docs/contributing/code-contribution-guide.md

```diff
@@ -322,6 +322,36 @@ const someFunction = (link: string, options: { prop1: string }): void => {
 };
 ```
 
+For function returns use:
```

Comment:
Suggest changing to: "For functions that return an object use:"

Reply:
Done 👍